src/HIE/Bios/Ghc/Api.hs

```diff
@@ -57,10 +57,10 @@ withGhcT body = do
 
 ----------------------------------------------------------------
 
--- | Initialise a GHC session by loading a given file into a given cradle.
+-- | Initialize a GHC session by loading a given file into a given cradle.
```

Comment:
😱

Reply:
Hehe, I actually wanted to ask: Are we using GB or US english here? Function names are US but most comments are GB